### PR TITLE
added CSS

### PR DESCRIPTION
**Pull request by pauldunkirk**

### CODE REVIEW

**Comment by pauldunkirk** [comment]
chapter 5 and 6 case study activities

**Comment by pauldunkirk** [comment]
fixed errors in anchor tags

File: activities [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (mbMosman) [inline activities]: You can delete this file, then do a `git add .` and it should be removed from GitHub too.

File: activities.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <nav>
+            <a href="index.html"> Home </a> &nbsp;
+            <a href="yurts.html"> Yurts </a> &nbsp;
+            <a href="activities.html"> Activities </a> &nbsp;
+            <a href="reservations.html"> Reservations </a>
+        </nav>
+        <div id="trailhero"> </div>
+        <main>
+            <h2> Activities at Pacific Trails</h2>
+            <h3><strong>Hiking</strong></h3>
```
Comment (mbMosman) [inline activities.html]: You can take out the strong tags now and add the bold using CSS.

File: pacific.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
+body {
+    font-family: Arial, Verdana, sans-serif;
+    background-color: #90C7E3;
+    background-image: linear-gradient(#FFFFFF, #90C7E3);
+    color: #666666;
+    background-repeat: no-repeat;
+}
+#wrapper {
+    width: 90%;
```
Comment (mbMosman) [inline pacific.css]: I believe this should be 80%

File: pacific.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    height: 60px;
+}
+main {
+    padding: 1px 20px 20px 20px; 
+    display: block;
+}
+h1 {
+    margin-bottom: 0;
+    margin-top: 0;
+    font-family: Georgia, Times New Roman, serif;
```
Comment (mbMosman) [inline pacific.css]: put quotes around multi-word fonts like "Times New Roman"

File: yurts.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+            <a href="index.html"> Home </a> &nbsp;
+            <a href="yurts.html"> Yurts </a> &nbsp;
+            <a href="activities.html"> Activities </a> &nbsp;
+            <a href="reservations.html"> Reservations </a>
+        </nav>
+        <div id="yurthero"> </div>
+        <main>
+            <h2> The Yurts at Pacific Trails</h2>
+
+            <h3><strong>What is a yurt?</strong></h3>
```
Comment (mbMosman) [inline yurts.html]: You can pull out the strong here too... just add the bold font in the CSS for the h3.

File: activities (hunk 1 above)
Comment (pauldunkirk) [inline activities]: delete what?

File: activities.html (hunk 2 above)
Comment (pauldunkirk) [inline activities.html]: I didn't read that in the book. Remember, I'm very literal. But I will take them out.

Comment (pauldunkirk) [inline activities.html]: ok. changed it.

File: activities (hunk 1 above)
Comment (pauldunkirk) [inline activities]: ok. thanks.

File: pacific.css (hunk 3 above)
Comment (pauldunkirk) [inline pacific.css]: changed it.

File: pacific.css (hunk 4 above)
Comment (pauldunkirk) [inline pacific.css]: They don't in the book :) 
But now I know. Thanks.

**Comment by mbMosman** [comment]
Great job!

**Pull request closed without merging**